utils/test/oo_config_tests.py

```diff
@@ -224,3 +226,82 @@ def test_load_host_no_master_or_node_specified(self):
             'public_hostname': 'a.example.com',
         }
         self.assertRaises(OOConfigInvalidHostError, Host, **yaml_props)
+
+    def test_inventory_file_quotes_node_labels(self):
+        """Verify a host entry wraps openshift_node_labels value in double quotes"""
+        yaml_props = {
+            'ip': '192.168.0.1',
+            'hostname': 'a.example.com',
+            'connect_to': 'a-private.example.com',
+            'public_ip': '192.168.0.1',
+            'public_hostname': 'a.example.com',
+            'new_host': True,
+            'roles': ['node'],
+            'node_labels': {
+                'region': 'infra'
+            },
+
+        }
+
+        new_node = Host(**yaml_props)
+        inventory = cStringIO.StringIO()
+        # This is what the 'write_host' function generates. write_host
+        # has no return value, it just writes directly to the file
+        # 'inventory' which in this test-case is a StringIO object
+        ooinstall.openshift_ansible.write_host(
+            new_node,
+            'node',
+            inventory,
+            schedulable=True)
+        # read the value of what was written to the inventory "file"
+        legacy_inventory_line = inventory.getvalue()
+
+        # Given the `yaml_props` above we should see a line like this:
+        #     openshift_node_labels="{'region': 'infra'}"
+        node_labels_expected = '''openshift_node_labels="{'region': 'infra'}"'''  # Quotes around the hash
+        node_labels_bad = '''openshift_node_labels={'region': 'infra'}'''  # No quotes around the hash
+
+        # The good line is present in the written inventory line
+        self.assertIn(node_labels_expected, legacy_inventory_line)
+        bad_line_present = node_labels_bad in legacy_inventory_line
+        # An unquoted version is not present
+        self.assertFalse(bad_line_present)
```

Comment:
Consider using `self.assertNotIn(node_labels_bad, legacy_inventory_line)` rather than using the `assertFalse` method for consistency.

Reference: https://docs.python.org/2/library/unittest.html#assert-methods